src/widget/position_widget.ts

```diff
@@ -1557,6 +1557,36 @@ class DimensionTool<Viewer extends object> extends Tool<Viewer> {
         }),
       ).element,
     );
+
+    this.registerDisposer(new MouseEventBinder(plot.element, inputEventMap));
+
+    registerActionListener<WheelEvent>(
```

Comment:
Can you factor out these handlers rather than duplicating them?

Reply:
@jbms I factorized the code in two dedicated private methods. The event action is passed as parameter of those methods. I can remove this logic from the methods and place it back inside the closure that calls the private methods.